src/event/special_user_message.rs

```diff
@@ -317,3 +318,50 @@ fn nab_command<'a>(
     }
     .boxed()
 }
+
+lazy_static::lazy_static! {
+    pub static ref DEPLOY_COMMAND: SpecialUserMessageTrigger = SpecialUserMessageTrigger {
+        regex: Regex::new("<@([A-z|0-9]+)> deploy (.*)").unwrap(),
+        then: &deploy_command,
+    };
+}
+fn deploy_command<'a>(
+    c: regex::Captures<'a>,
+    _: Message,
+    _: &'a Sender<FarmingInputEvent>,
+) -> HandlerOutput<'a> {
+    use std::collections::HashMap;
+    use std::fs;
+    async move {
+        let deploy_target = c
+            .get(2)
+            .ok_or_else(|| "No deploy target".to_string())?
+            .as_str()
+            .to_string();
+        let manifest = fs::read_to_string(format!("./manifests/{}.yml", deploy_target))
+            .map_err(|_| "Invalid deploy target".to_string())?;
+        let mut json_request: HashMap<String, String> = HashMap::new();
```

Comment:
@muirrum finally had a chance to look at that PR, is there any particular reason you construct the JSON request like this
```rs
        let mut json_request: HashMap<String, String> = HashMap::new();
        json_request.insert("manifest".to_string(), manifest);
        json_request.insert(
            "note".to_string(),
            "Submitted automatically by hackagotchi".to_string(),
        );
instead of just doing
```rs
let json_request = json!({
    "manifest": manifest,
    "note": "Submitted automatically via hackagotchi",
});
```
or even supplying that `json!` macro call directly to the `.json()` method you call on your `reqwest::ResponseBuilder`.